README.md

```diff
@@ -63,6 +63,16 @@ This script does the following steps for you:
 - create a TAB (tock application bundle)
 - if you have a J-Link compatible board connected: flash this TAB to your board (using tockloader)
 
+### Enabling rust-embedded support
+
+libtock-c can be built to be compatible with the rust-embedded
+[embedded_hal](https://docs.rs/embedded-hal/1.0.0/embedded_hal/index.html) by
+including the following when running `make`
+
+```shell
+FEATURES=rust_embedded
+```
```

Comment:
I'm having trouble seeing the utility of this section. `libtock-rs` isn't really a binary, so building it on its own isn't useful. Building it with the `rust_embedded` feature as a dependency of another crate is useful, but that isn't normally done using `libtock-rs`' Makefile.

Mentioning the `cargo` feature `rust_embedded` probably makes sense though.

Reply:
One part is just to allow testing the feature directly in libtock-rs. It also just feels like a easy way to describe what it does

I added a comment saying that the feature can be enabled via Cargo as well